app/assets/stylesheets/juntos_bootstrap/_colors.sass

```diff
@@ -6,6 +6,7 @@ $gray-one: #F1F4F4
 $gray-two: #F1F2F3
 $border-gray: #E9E9EC
 $border-strong: #DDDDDD
+$border-yellow: #FFB74D
 
 $blue-one: #3F4752
 $blue-two: #79808B
```

Comment:
Sei que esses nomes já estavam aqui, mas eles não dizem muito sobre o que tá rolando. Se quiserem, pode, usar algo como o [Name that Color](http://chir.ag/projects/name-that-color)